audio/freac/Portfile

```diff
@@ -25,6 +25,21 @@ checksums           rmd160  2f1f9ea871793cec1a8fe02a96fb9c03b1fa0c49 \
 depends_lib         port:boca \
                     port:smooth
 
+depends_run         port:lame \
+                    port:ffmpeg \
```

Comment:
Can `ffmpeg-devel` also be used? If so, use this specifications to allow it:

`path:lib/libavcodec.dylib:ffmpeg`

Reply:
Yes. I tested it and ffmpeg-devel works as well.